public/app/plugins/datasource/graphite/configuration/ConfigEditor.tsx

```diff
@@ -77,48 +84,52 @@ export class ConfigEditor extends PureComponent<Props, State> {
           onChange={onOptionsChange}
           secureSocksDSProxyEnabled={config.secureSocksDSProxyEnabled}
         />
-        <h3 className="page-heading">Graphite details</h3>
-        <div className="gf-form-group">
-          <div className="gf-form-inline">
-            <div className="gf-form">
-              <InlineFormLabel tooltip="This option controls what functions are available in the Graphite query editor.">
-                Version
-              </InlineFormLabel>
+        <FieldSet>
+          <legend className="page-heading">Graphite details</legend>
```

Comment:
I am using the class here, instead of FieldSet's label, because they have different font sizes.